command/secrets_enable_test.go

```diff
@@ -201,9 +203,8 @@ func TestSecretsEnableCommand_Run(t *testing.T) {
 			}
 		}
 
-		// Removing one from logical list since plugin is a virtual backend
-		if len(backends) != len(logicalBackends)-1 {
-			t.Fatalf("expected %d logical backends, got %d", len(logicalBackends)-1, len(backends))
+		if len(backends) != len(builtinplugins.Registry.Keys(consts.PluginTypeSecrets))+2 {
```

Comment:
Can you put a comment on why this is a +2 now?

Reply:
Yes, good idea! https://github.com/hashicorp/vault/pull/5536/commits/4b414a85f3a167747ada39e1feadcc849f917748